backend/src/main/java/com/team4/backend/service/StudentService.java

```diff
@@ -98,4 +100,14 @@ public StudentService(StudentRepository studentRepository, PBKDF2Encoder pbkdf2E
                         Mono.error(new UserNotFoundException("Could not find student with id: " + studentId))
                 );
     }
+
+    public Mono<Long> updateStudentStateForAllStudentThatInterviewDateHasPassedWeb() {
+        return studentRepository.findAllByStudentStateAndInterviewsDateIsNotEmpty(StudentState.INTERNSHIP_NOT_FOUND)
+                .map(student -> {
```

Comment:
Shouldn't it be flatMap as `findAllByStudentStateAndInterviewsDateIsNotEmpty` returns Flux ?

Reply:
no need